pineapple-core/src/main/java/sh/miles/pineapple/config/adapter/ColorAdapter.java

```diff
@@ -0,0 +1,25 @@
+package sh.miles.pineapple.config.adapter;
+
+import org.bukkit.Color;
```

Comment:
Use BungeeChat color org.bukkit.Color is as good as deprecated and can be retrieved from a BungeeChat color where needed